code/src/Components/ThoughtsForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const ThoughtsForm = ({ newThought, onNewThoughtChange, onThoughtSubmit }) => {
+   
+    return (
+        <form onSubmit={onThoughtSubmit}>
+            <div className="greetingcard">
+            <h1> What's making you happy right now?</h1>
+            <textarea placeholder="Minimum 6 characters"value={newThought} onChange={onNewThoughtChange}/>
+            <div className="main">
+            <button className="button" type="submit" disabled={newThought.length < 6 || newThought.length > 140}><span role="img" aria-label="heartemoji">❤️</span>Send Happy Thought!<span role="img" aria-label="heartemoji">❤️</span></button></div></div>
```

Comment:
Nicely done with the disabled function, you can actually style` button:disabled` in your cssfile to if you want to make it extra clear